util/vagrant/Vagrantfile

```
@@ -0,0 +1,23 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
```

```suggestion
  config.vm.box = "bento/ubuntu-18.04"
```

:+1: on @arjenroodselaar 's suggestion.